internal/proto/milvus.proto

```diff
@@ -97,6 +118,7 @@ message DescribeCollectionResponse {
   repeated string physical_channel_names = 5;
   uint64 created_timestamp = 6; // hybrid timestamp
   uint64 created_utc_timestamp = 7; // physical timestamp
+  repeated string aliases = 8;
```

Comment:
Seems this member is defined but not used?

Reply:
I was planning to add use this field in separate PR, since this PR is already getting big.